LocalMultiplayerAgent/DeploymentTool/readme.md

```diff
@@ -0,0 +1,57 @@
+﻿After testing their game server(s) on LMA, you may want to go ahead and deploy them.
+This tool was developed exactly for this purpose; allow you go ahead to deploy your game servers.
+Technically, you can also just run deployment without testing with LMA **but testing is highly recommended**
+
+## Requirements:
+
+- Make sure your [MultiplayerSettings.json](../MultiplayerSettings.json) is configured right
+    - These parameters in MultiplayerSettings.json are not required/do not matter if you are trying to run deployment:
+    `  
+    "OutputFolder": "",
+    "NumHeartBeatsForActivateResponse": 5,
+    "NumHeartBeatsForTerminateResponse": 10,
+    "AgentListeningPort": 56001,
+    "BuildId": "",
+    "Region": "",
+    "SessionConfig": {
+        "SessionId": "ba67d671-512a-4e7d-a38c-2329ce181946",
+        "SessionCookie": null,
+        "InitialPlayers": [ "Player1", "Player2" ]
+    }
+    `
+
+## Steps:
+
+- Configure [build settings](./DeploymentSettings.json). Feel free to reconfigure DeploymentSettings.json to suit your needs. Example:
```

Comment:
We can provide examples of DeploymentSettings.json along with MultiplayerSettings.json for different builds - Linux Container, Windows Container and Windows Process